app/create_app.py

```diff
@@ -140,9 +141,24 @@ def find_fund_in_request():
             fund = get_fund_data(request.args.get("fund_id"), as_dict=True)
         elif request.args.get("fund"):
             fund = get_fund_data_by_short_name(request.args.get("fund"))
+        elif (
+            application_id := request.args.get("application_id")
+            or request.view_args.get("application_id")
+            or request.form.get("application_id")
+        ):
+            application_id = (
+                request.args.get("application_id")
+                or request.view_args.get("application_id")
+                or request.form["application_id"]
+            )
```

Comment:
Also I think you need to change the order:

```python
application_id := (
    request.args.get("application_id")
    or request.view_args.get("application_id")
    or request.form.get("application_id")
)
```

Reply:
Maybe not actually... sorry for overcomplicating

Reply:
> This is no longer needed since we're doing
> 
> ```python
> application_id := request.args.get("application_id") or request.view_args.get("application_id") or request.form.get("application_id")
> ```
> 
> https://docs.python.org/3/whatsnew/3.8.html#assignment-expressions

Ahh. I removed it but forgot to commit. My bad